library/core/src/num/f128.rs

```diff
@@ -829,8 +828,8 @@ impl f128 {
     /// ```
     #[inline]
     #[unstable(feature = "f128", issue = "116909")]
-    // #[unstable(feature = "num_midpoint", issue = "110840")]
-    pub fn midpoint(self, other: f128) -> f128 {
+    #[rustc_const_unstable(feature = "f128", issue = "116909")]
+    pub const fn midpoint(self, other: f128) -> f128 {
```

Comment:
I believe you don't need a separate `rustc_const_unstable` gate anymore if the whole thing is `unstable`, just the `const` keyword (applies to `f16` too)